### ISSUE TITLE
Model accuracy drops after export

### ISSUE THREAD

**Issue by Fradd747** [PaddlePaddle/PaddleOCR#15141]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

I trained a detection model (following instructions [here](https://paddlepaddle.github.io/PaddleOCR/main/en/ppocr/model_train/detection.html)) on top of pretrain `ResNet50_vd_ssld_pretrained`. When I run a test via 
`python PaddleOCR/tools/infer_det.py -c model_output/det_r50_vd_db.yml -o Global.infer_img="./det_test_set/6Db7UW8nDzgoYqsDKBZsOaK68uCxzZG6ncFTcOhO.jpeg" Global.pretrained_model="./model_output/best_accuracy"`
everything works just fine and image is correctly marked with detection box. I tried it on 20 images, all of them were marked correctly.

But when I export the model using
`python PaddleOCR/tools/export_model.py -c model_output/det_r50_vd_db.yml -o Global.pretrained_model="./model_output/best_accuracy" Global.save_inference_dir="./output/det_db_inference/"`
and then use this model via PaddleOCR python library:
```
ocr = PaddleOCR(
    det_model_dir='output/det_db_inference',
    rec_model_dir='models/ch_PP-OCRv4_rec_infer',
    cls_model_dir='models/ch_ppocr_mobile_v2.0_cls_infer',
    use_angle_cls=True,
    use_gpu=True,
)
result = ocr.ocr(image_path, det=True, rec=False, cls=True)
```
accuracy significantly drops. Correctly marking only 4/20 images. Why is this happening?

`det_r50_vd_db.yml`:
```
Global:
  use_gpu: true
  epoch_num: 120
  log_smooth_window: 20
  print_batch_step: 20
  save_model_dir: ./output/det_r50_vd_vin/
  save_inference_dir: ./output/det_r50_vd_vin/
  save_epoch_step: 10
  # evaluation is run every 2000 iterations
  eval_batch_step: [0,54]
  cal_metric_during_train: False
  pretrained_model: ./pretrain/ResNet50_vd_ssld_pretrained
  use_visualdl: True
  infer_img: train_data/det/valid/images/image_1020.jpg
  save_res_path: ./output/det_db_res50_vin/predicts_db.txt
  #use_amp: True

Architecture:
  model_type: det
  algorithm: DB
  Transform:
  Backbone:
    name: ResNet_vd
    layers: 50
  Neck:
    name: DBFPN
    out_channels: 256
  Head:
    name: DBHead
    k: 50

Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 1.5

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    learning_rate: 0.0015
    decay: Cosine
    warmup_epoch: 3
  regularizer:
    name: 'L2'
    factor: 0.0001

PostProcess:
  name: DBPostProcess
  thresh: 0.3
  box_thresh: 0.5
  max_candidates: 1000
  unclip_ratio: 1.5

Metric:
  name: DetMetric
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/det/train/images
    label_file_list:
      - ./train_data/det/train/labels_ocr.txt
    ratio_list: [1.0]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - IaaAugment:
          augmenter_args:
            #- { 'type': Fliplr, 'args': { 'p': 0.5 } }
            #- { 'type': Affine, 'args': { 'rotate': [-15, 15] } }
            - { 'type': MotionBlur, 'args': { 'k': 3 } }
            - { 'type': RandomBrightnessContrast, 'args': { 'brightness_limit': 0.2, 'contrast_limit': 0.2, 'p': 0.5 } }
            #- { 'type': Resize, 'args': { 'size': [0.5, 3] } }
      - EastRandomCropData:
          size: [736, 736]
          max_tries: 50
          keep_ratio: true
      - MakeBorderMap:
          shrink_ratio: 0.4
          thresh_min: 0.3
          thresh_max: 0.7
      - MakeShrinkMap:
          shrink_ratio: 0.4
          min_text_size: 8
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'threshold_map', 'threshold_mask', 'shrink_map', 'shrink_mask'] # the order of the dataloader list
  loader:
    shuffle: True
    drop_last: False
    batch_size_per_card: 56
    num_workers: 12

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/det/valid/images
    label_file_list:
      - ./train_data/det/valid/labels_ocr.txt
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - DetResizeForTest:
          image_shape: [736, 736]
          keep_ratio: true
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'shape', 'polys', 'ignore_tags']
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1 # must be 1
    num_workers: 12

EarlyStopping:
  monitor: hmean
  patience: 7
  mode: max

```

I also tried to use this script
`python PaddleOCR/tools/infer/predict_det.py --det_model_dir="./output/det_db_inference/" --image_dir="./det_test_set/" --use_gpu=True`
and it works just right.

### 🏃‍♂️ Environment (运行环境)

Windows 11,
PaddleOCR 2.10.0

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

Model `best accuracy`, `exported model`, `config file` and `test image` can be found [here](https://drive.google.com/drive/folders/1UhGimx3MTtd9KTdNoXl3q3KEHgmVIvej?usp=sharing).
```
ocr = PaddleOCR(
    det_model_dir='models/det/det_db_inference',
    rec_model_dir='models/ch_PP-OCRv4_rec_infer', #automatically downloads
    cls_model_dir='models/ch_ppocr_mobile_v2.0_cls_infer', #automatically downloads
    use_angle_cls=True,
    use_gpu=True,
)
result = ocr.ocr("test_image.jpeg", det=True, rec=False, cls=True)
print(result)
```

**Comment by Fradd747** [comment]
Hey, for anyone who has encountered this issue, here's my solution. When you input your image path into the PaddleOCR library `result = ocr_engine.ocr(image_path, det=True, rec=False, cls=True)`, it doesn't automatically rotate the image if image data are stored in one direction and image orientation is stored in EXIF data. You can check if your image has an EXIF tag using any online site like [https://jimpl.com/](https://jimpl.com/) and finding the "EXIF" tag. So in other words, the image was sent sideways to the OCR function, and detection couldn't be processed (because of that shift).

The solution for this is to open the image using `Image.open()`, rotate it via EXIF tag, and then convert it to a NumPy array (acceptable `ocr()` fn input parameter). The result should be now okay. If you try to draw a detection area in the image, don't forget to also rotate the image you're drawing the area in.

Pillow does not automatically apply this transformation to the data. However, it has a method to do so - https://pillow.readthedocs.io/en/stable/reference/ImageOps.html#PIL.ImageOps.exif_transpose

Example code that solved my issue:
```
from paddleocr import PaddleOCR
from PIL import Image, ImageDraw, ImageOps
import numpy as np

image_path = 'images/testing_image.jpg'
image = Image.open(image_path)
image = ImageOps.exif_transpose(image) # Apply rotation
image_np = np.array(image) # Convert to Numpy array

result = ocr_engine.ocr(image_np, det=True, rec=False, cls=True)
draw = ImageDraw.Draw(image) # Using already rotated image
draw.polygon(box_info, outline='red', width=2)
image.show()
```
